src/main/resources/camel-project-template/pom.xml

```diff
@@ -117,4 +117,108 @@
     </plugins>
   </build>
 
+<repositories>
+    <repository>
+        <id>fuse-public-repository</id>
+        <name>FuseSource Community Release Repository</name>
```

Comment:
is the fusesource repository really needed? If not, it will be best to get rid of it. (and if it is maybe we can try to report to get rid of it one day ^^)

Reply:
you need to ask to Fuse runtime team

Reply:
Waiting for Fuse 7.1 to be released - but updated repositories and as soon as that productized version is available we're good to go.